packages/design-system-react/package.json

```diff
@@ -48,14 +48,24 @@
   },
   "devDependencies": {
     "@metamask/auto-changelog": "^3.4.4",
+    "@storybook/react": "^8.3.5",
+    "@storybook/test": "^8.3.5",
     "@types/jest": "^27.4.1",
+    "@types/react": "^18.2.0",
+    "@types/react-dom": "^18.2.0",
     "deepmerge": "^4.2.2",
     "jest": "^27.5.1",
     "ts-jest": "^27.1.4",
     "typedoc": "^0.24.8",
     "typedoc-plugin-missing-exports": "^2.0.0",
     "typescript": "~5.2.2"
   },
+  "peerDependencies": {
+    "@metamask/design-system-tailwind-preset": "^0.0.0",
+    "react": "^16.0.0",
+    "react-dom": "^16.0.0",
+    "tailwindcss": "^3.0.0"
```

Comment:
Aligns with tailwind version across repository and [portfolio minimum major version](https://github.com/consensys-vertical-apps/metamask-portfolio/blob/main/package.json#L90)

<img width="303" alt="Screenshot 2024-10-30 at 2 36 13 PM" src="https://github.com/user-attachments/assets/d6889057-f6a3-49a1-9025-5900d4da827f">